settings.gradle

```diff
@@ -13,7 +13,7 @@ exec {
 dependencyResolutionManagement {
     versionCatalogs {
         libs {
-            version("checkerFramework", "3.42.0-eisop4")
+            version("checkerFramework", "3.42.0-eisop5")
```

Comment:
Would it make sense to also make `initialize-project` stop cloning `checker-framework`, since that Checker Framework wouldn't be used by default?

Reply:
It looks like `initialize-project` is only [cloning](https://github.com/jspecify/jspecify-reference-checker/blob/27d2334f1c94a223a70c0c98d11058dadce38d94/initialize-project#L71) `jdk`? Am I missing how that script works?

Reply:
Oops, I was looking [at `main`](https://github.com/jspecify/jspecify-reference-checker/blob/a9958c5cc11906b16027e5d105501216d52b3bcc/initialize-project#L68) and not at `main-eisop`. Never mind.